flambe/nn/module.py

```diff
@@ -120,3 +122,44 @@ def clip_gradient_norm(self, threshold: float):
         # Only compute over parameters that are being trained
         parameters = filter(lambda p: p.requires_grad and p.grad is not None, self.parameters())
         nn.utils.clip_grad_norm_(parameters, threshold)
+
+
+class Encoder(Module):
```

Comment:
I think this name is a little confusing because of our history using this word for different things.

Reply:
Suggestions?